### ISSUE TITLE
Runaway argument in LaTeX 

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#587]
I get this error with tables in 1.0.6 

```
Chapter 1.
(/usr/local/texlive/2008/texmf-dist/tex/latex/psnfss/ts1ptm.fd)
Runaway argument?
{ Folge 
! Paragraph ended before \text@command was complete.
<to be read again> 
                   \par 
l.223 \end{tabulary}
```

See attached minimal project.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/587
- Originally reported by: Jan Ulrich Hasecke
- Originally created at: 2011-01-06T20:10:07.242

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-01-06 19:11:21+00:00_

Fixed in <<cset 0908c23ef067d7b946e7e3fbe9847f86fea582ce>>.

**Comment by shimizukawa** [comment]
_From Jan Ulrich Hasecke on 2011-01-06 19:14:29+00:00_

Wow, that was fast ;-) I saw this commit, but don't want to try out trunk. Thx.
